website/source/docs/builders/ncloud.html.md

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+    As Packer allows users to develop a custom builder as a plugin, NAVER CLOUD PLATFORM provides its own Packer builder for your convenience.
+You can use NAVER CLOUD PLATFORM's Packer builder to easily create your server images.
+layout: docs
+page_title: 'Naver Cloud Platform - Builders'
+sidebar_current: 'docs-builders-ncloud'
+---
+
+# NAVER CLOUD PLATFORM Builder
+
+As Packer allows users to develop a custom builder as a plugin, NAVER CLOUD PLATFORM provides its own Packer builder for your convenience.
+You can use NAVER CLOUD PLATFORM's Packer builder to easily create your server images.
+
+#### Sample code of template.json
+
+```
+{
+  "variables": {
+    "ncloud_access_key": "FRxhOQRNjKVMqIz3sRLY",
+    "ncloud_secret_key": "xd6kTO5iNcLookBx0D8TDKmpLj2ikxqEhc06MQD2"
+  },
+  "builders": [
+    {
+      "type": "ncloud",
+      "access_key": "{{user `ncloud_access_key`}}",
+      "secret_key": "{{user `ncloud_secret_key`}}",
+
+      "server_image_product_code": "SPSW0WINNT000016",
+      "server_product_code": "SPSVRSSD00000011",
+      "member_server_image_no": "4223",
+      "server_image_name": "packer-test {{timestamp}}",
+      "server_description": "server description",
+      "user_data": "CreateObject(\"WScript.Shell\").run(\"cmd.exe /c powershell Set-ExecutionPolicy RemoteSigned & winrm quickconfig -q & sc config WinRM start= auto & winrm set winrm/config/service/auth @{Basic=\"\"true\"\"} & winrm set winrm/config/service @{AllowUnencrypted=\"\"true\"\"} & winrm get winrm/config/service\")",
+      "region": "US-West"
+    }
+  ]
+}
+```
+
+#### Description
```

Comment:
Please split this into a `required` and `optional` section.

Configuration options should have the format of

```
*   `option_name` (type) - Description. Default Value.

* ...
```